streamlit_app.py

```diff
@@ -58,7 +58,12 @@
     highlight_figures = st.toggle('Figures and tables', value=True, disabled=not st.session_state['uploaded'])
     highlight_callout = st.toggle('References citations in text', value=True, disabled=not st.session_state['uploaded'])
     highlight_citations = st.toggle('Citations', value=True, disabled=not st.session_state['uploaded'])
-    st.divider()
+
+    st.header("Page Selection")
+    page_options = list(range(1, 101))  # Replace 101 with the actual number of pages + 1
```

Comment:
I think this line indicates the problem with this feature... 🤔

Reply:
I agree with your point. However, as a parameter for the library, it is an Optional List type and can be passed in the form of `[1, 2, 3]`. How best to show this select box in the Structure-viewer? Another idea I'm considering is to allow users to input arrays as strings. Do you have any other ideas?

Reply:
No ideas, without having the pdf in advance is not possible to guess any page range